src/main/java/database/users.csv

```diff
@@ -0,0 +1 @@
+ userID,isAdmin,email,password,listPosts,listFavourites;
```

Comment:
I think the semicolon at the end is unnecessary and may mess with reading the csv.